projects/plugins/jetpack/extensions/shared/components/media-player-control/index.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarGroup, ToolbarButton, RangeControl } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import {
+	ControlBackFiveIcon,
+	ControlForwardFiveIcon,
+	ControlSyncIcon,
+	ControlUnsyncIcon,
+} from '../../icons';
+import { STATE_PAUSED, STATE_PLAYING, STORE_ID } from '../../../store/media-source/constants';
+import { convertSecondsToTimeCode } from './utils';
+
+function noop () {}
+
+export function MediaPlayerControl( {
+	skipForwardTime = 5,
+	jumpBackTime = 5,
+	customTimeToPlay,
+	syncMode,
+	onSyncModeToggle = noop,
+	playIcon = 'controls-play',
+	pauseIcon = 'controls-pause',
+	backFiveIcon = ControlBackFiveIcon,
+	forwardFiveIcon = ControlForwardFiveIcon,
+	onTimeChange = noop,
```

Comment:
Should this be implemented now? 🙂

Reply:
It is/ `noop` is the default value of the `onTimeChange` property.